integration/ramdisk/ramdisk.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+cidir=$(dirname "$0")
+
+source "${cidir}/../../lib/common.bash"
+
+set -e
+
+# Environment variables
+IMAGE="${IMAGE:-busybox}"
+CONTAINER_NAME="${CONTAINER_NAME:-test}"
+PAYLOAD_ARGS="${PAYLOAD_ARGS:-sleep 30}"
+
+# Set the runtime if not set already
+RUNTIME="${RUNTIME:-kata-runtime}"
+
+# File to save ramdisk results
+TMP_FILE=$(mktemp ramdisk.XXXXXXXXXX)
+
+remove_tmp_file() {
+	rm -rf $TMP_FILE
+}
+
+trap remove_tmp_file EXIT
+
+setup() {
+	clean_env
+
+	# Check that processes are not running
+	check_processes
+
+	# Stop docker
+	sudo -E systemctl stop docker
+}
+
+test_ramdisk() {
+	# Enable ramdisk with dockerd
+	(sudo -E DOCKER_RAMDISK=true dockerd -D --add-runtime ${RUNTIME}=${RUNTIME_PATH} 2>/dev/null) &
+
+	# Run container
+	docker run --rm -d --name=${CONTAINER_NAME} --runtime=${RUNTIME} ${IMAGE} sh -c ${PAYLOAD_ARGS}
+
+	# Extract the logs of kata-runtime
+	sudo journalctl -t ${RUNTIME} > ${TMP_FILE}
```

Comment:
It might be an idea to limit this journalctl to a just the entries generated by this test. Maybe we can do that by using `date` to grab a start time, and then use a `--since` on the journalctl to grab only entries generated since this test started.
I know we don't have afaik any other tests that will place a no-pivot into the logs, but you never know ;-)